tests/rules/method.js

```diff
@@ -219,6 +220,49 @@ eslintTester.run("method", rule, {
             code: "(e = node.insertAdjacentHTML('beforebegin', '<s>safe</s>'))()",
             parserOptions: { ecmaVersion: 6 },
         },
+
+        // Typescript support tests
+        // raw strings can be assigned to innerHTML
+        {
+            code: "(options as HTMLElement).innerHTML = '<s>safe</s>';",
```

Comment:
Tests for .innerHTML should go into test/rules/property.js

Reply:
done